test_file.py

```diff
@@ -0,0 +1 @@
+print(file not empty)
```

Comment:
print statement will not work, since it doesn't have quotes, please change to the following:

```
print("file not empty")
```